packages/flutter_test/lib/src/binding.dart

```diff
@@ -1919,9 +1919,9 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
 
   @override
   ViewConfiguration createViewConfiguration() {
-    return TestViewConfiguration(
+    return TestViewConfiguration.fromView(
       size: _surfaceSize ?? _kDefaultTestViewportSize,
-      window: window,
+      view: window,
```

Comment:
Would it be reasonable to change this to `platformDispatcher.implicitView` while we're here, or is that planned for a separate PR?

Reply:
I am punting on updating `createViewConfiguration` overrides until I have updated the base class implementation to do the right thing. `createViewConfiguration`'s role will change significantly with actual multi view support in the framework.

Reply:
Sounds good, thanks for explaining!